calicoctl/commands/delete.go

```diff
@@ -16,7 +16,8 @@ package commands
 
 import (
 	"fmt"
-	"strings"
+  "strings"
```

Comment:
Looks like you'll need to run `gofmt` on this file

Reply:
I Love `gofmt`
Thank you!